app/components/primer/alpha/navigation_list.html.erb

```diff
@@ -0,0 +1,18 @@
+<action-list>
+  <%= render(Primer::BaseComponent.new(tag: :nav, **@nav_arguments)) do %>
+    <%= render(Primer::BaseComponent.new(tag: :ul, **@list_arguments)) do %>
+      <% items.each do |item| %>
+        <%= item %>
+      <% end %>
+      <% unless sections.empty? %>
+        <li class="ActionList-sectionDivider"></li>
+      <% end %>
+      <% sections.each_with_index do |section, idx| %>
+        <% if idx > 0 %>
+          <li class="ActionList-sectionDivider"></li>
+        <% end %>
```

Comment:
would this be the same as

```erb
<% sections.each_with_index do |section, idx| %>
  <li class="ActionList-sectionDivider"></li>
```

Reply:
This is present to make sure the first section doesn't have a leading divider.